lib/streamlit/elements/markdown.py

```diff
@@ -209,6 +209,24 @@ def latex(self, body: Union[SupportsStr, "sympy.Expr"]) -> "DeltaGenerator":
         latex_proto.element_type = MarkdownProto.Type.LATEX
         return self.dg._enqueue("markdown", latex_proto)
 
+    @gather_metrics("divider")
+    def divider(self) -> "DeltaGenerator":
+        """Display horizontal ruler.
+
+        (This is a convenience wrapper around `st.markdown()`)
+
+        Example
+        -------
+        >>> import streamlit as st
+        >>>
+        >>> st.divider()
+
+        """
+        divider_proto = MarkdownProto()
```

Comment:
In general I'm not sure if this widget should be based on Markdown because we lose the ability to style it. I was more thinking about a dedicated React component.

Reply:
The product said it should be a wrapper on st.markdown and now we even have it explicitly mentioned in the method documentation.